test_cromwell.sh

```diff
@@ -11,51 +11,50 @@ trap "shutdown" EXIT
 
 set -e
 
-EXIT_CODE=31
+EXIT_CODE=1
```

Comment:
Just curious, why this change ? Also what is it ? The exit code of this script ?